roachpb/api.proto

```diff
@@ -578,8 +578,10 @@ message Header {
   // performed. If the timestamp is set to zero value, its value
   // is initialized to the wall time of the receiving node.
   optional Timestamp timestamp = 1 [(gogoproto.nullable) = false];
-  // cmd_id is optionally specified for request idempotence
-  // (i.e. replay protection).
+  // replica specifies the location where the request was sent to.
```

Comment:
> // replica specifies the destination of the request.

doesn't seem worthwhile to have internal implementation details on the proto comments.